docs/guides/rule-evaluations-via-query-frontend.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Rule evaluations via query frontend"
+linkTitle: "Rule evaluations via query frontend"
+weight: 10
+slug: rule-evalutions-via-query-frontend
+---
+
+This guide explains how to configure the Ruler to evaluate rules via Query Frontends instead of the Ingesters and the pros and cons of rule evaluation via Query Frontend.
+
+## How to enable
+
+By default, the Ruler queries to the Ingesters for evaluating rules (alerting rules or recording rules). If you have set `-ruler.frontend-address` then the Ruler query to the Query Frontend for evaluation rules.
```

Comment:
the Ruler queries the Ingesters.
the Ruler queries the Query Frontend.

Ruler queries both Ingesters and Store Gateway depending on the Rule time range.